lib/datadog/lambda/utils/extension.rb

```diff
@@ -12,12 +12,14 @@
 module Datadog
   # Utils contains utility functions shared between modules
   module Utils
-    AGENT_URL = 'http://127.0.0.1:8124'
-    HELLO_PATH = '/lambda/hello'
```

Comment:
theres no use for hello anymore right?

Reply:
Correct, that route is misleading. Since it was used to tell the extension that the tracer would be handling everything (including inferred spans).